### PR TITLE
Make up JSDoc for codes added/changed during the lite version of contact information development

### PR DESCRIPTION
**Pull request by eason9487**
### Changes proposed in this Pull Request:

Make up JSDoc for codes added/changed during the lite version of contact information development.

### Detailed test instructions:

Only JSDoc were added. Please checkout diff to see if any docs are unclear, unreadable, or confused.

### Changelog entry

>

### CODE REVIEW

File: js/src/components/contact-information/index.js [hunk 1]
```diff
@@ -72,6 +76,17 @@ export function ContactInformationPreview() {
...
+/**
+ * @typedef { import("./phone-number-card").onPhoneNumberChange } onPhoneNumberChange
+ */
+
+/**
+ * Renders a contact information section with specified initial state and texts, determined by the `view` prop.
+ *
+ * @param {Object} props React props.
+ * @param {'setup-mc'|'settings'} props.view Indicate where this component is used.
+ * @param {onPhoneNumberChange} [props.onPhoneNumberChange] Called when phone number are changed.
```
Comment (ecgan) [inline js/src/components/contact-information/index.js]:
```suggestion
 * @param {onPhoneNumberChange} [props.onPhoneNumberChange] Called when phone number is changed.
```

File: js/src/components/contact-information/no-contact-information-card.js [hunk 2]
```diff
@@ -12,6 +12,14 @@ import Section from '.~/wcdl/section';
 import AppButton from '.~/components/app-button';
 import './no-contact-information-card.scss';
 
+/**
+ * Renders a warning section to prompt users to add contact information.
+ *
+ * @param {Object} props React props.
+ * @param {Function} props.onEditClick Called when clicking on the "Add information" button.
+ * @param {string} props.learnMoreUrl The URL of learn more link and the props `href` value of the link's click tracking event.
+ * @param {string} props.learnMoreLinkId The props `link_id` value of the learn more link's click tracking event.
```
Comment (ecgan) [inline js/src/components/contact-information/no-contact-information-card.js]:
```suggestion
 * @param {string} props.learnMoreUrl Value for the "Learn more" link URL and the click tracking `href` event property.
 * @param {string} props.learnMoreLinkId Value for the click tracking `link_id` event property.
```

File: js/src/components/vertical-gap-layout/index.js [hunk 3]
```diff
@@ -13,6 +13,15 @@ const sizeClassName = {
 	overlap: 'gla-vertical-gap-layout__overlap',
 };
 
+/**
+ * Renders a wrapper layout to make its children to be aligned vertically with a specified gap.
+ *
+ * @param {Object} props React props.
+ * @param {string} [props.className] Additional CSS class name to be appended.
+ * @param {''|'large'|'overlap'} [props.size=''] Indicate the gap between children. The empty string as default value is a normal gap.
+ *     'large': A larger gap.
```
Comment (ecgan) [inline js/src/components/vertical-gap-layout/index.js]: 💅 🤔  Think it might be good to specify the value for normal gap and large gap.

**Review by ecgan — APPROVED** [review]
LGTM 👍  Added a few suggestions in comments.

File: js/src/components/vertical-gap-layout/index.js (hunk 3 above)
Comment (eason9487) [reply js/src/components/vertical-gap-layout/index.js]: Adjusted by https://github.com/woocommerce/google-listings-and-ads/pull/968/commits/9baa3ebae5eda8aebcb0e528aa4d408472dac26e.

**Pull request merged**